airbyte-integrations/connectors/destination-iceberg/src/test-integration/java/io/airbyte/integrations/destination/iceberg/container/NessieServerWithMinioCompose.java

```diff
@@ -0,0 +1,108 @@
+import static io.airbyte.integrations.destination.iceberg.IcebergConstants.NESSIE_CATALOG_URI_CONFIG_KEY;
+import static io.airbyte.integrations.destination.iceberg.IcebergConstants.S3_ACCESS_KEY_ID_CONFIG_KEY;
+import static io.airbyte.integrations.destination.iceberg.IcebergConstants.S3_BUCKET_REGION_CONFIG_KEY;
+import static io.airbyte.integrations.destination.iceberg.IcebergConstants.S3_ENDPOINT_CONFIG_KEY;
+import static io.airbyte.integrations.destination.iceberg.IcebergConstants.S3_SECRET_KEY_CONFIG_KEY;
+import static io.airbyte.integrations.destination.iceberg.IcebergConstants.S3_WAREHOUSE_URI_CONFIG_KEY;
+import static java.util.Map.entry;
+import static java.util.Map.ofEntries;
+
+import java.nio.file.Path;
+import java.time.Duration;
+import java.util.Map;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.integrations.destination.iceberg.config.format.DataFileFormat;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class NessieServerWithMinioCompose extends DockerComposeContainer<NessieServerWithMinioCompose> {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(NessieServerWithMinioCompose.class);
+  private static final String LOCAL_RELATIVE_PATH = "src/test-integration/resources/";
+  private static final String COMPOSE_PATH = LOCAL_RELATIVE_PATH + "nessie-catalog-compose.yml";
+  private static final int NESSIE_SERVER_PORT = 19120;
+  private static final int MINIO_PORT = 9000;
+  private static final String NESSIE_SERVICE_NAME = "nessie_1";
+  private static final String MINIO_SERVICE_NAME = "minio_1";
+
+  public NessieServerWithMinioCompose() {
+    super(Path.of(COMPOSE_PATH).toFile());
+    super.withExposedService(NESSIE_SERVICE_NAME,
+                    NESSIE_SERVER_PORT,
+        Wait.forListeningPort().withStartupTimeout(Duration.ofSeconds(30)))
+            .withExposedService(MINIO_SERVICE_NAME,
+                MinioContainer.MINIO_PORT,
+                Wait.forHttp(MinioContainer.HEALTH_ENDPOINT).withStartupTimeout(Duration.ofSeconds(60)))
+            .withLocalCompose(true);
+  }
+
+  @Override
+  public void start() {
+    long startTime = System.currentTimeMillis();
+    super.start();
+    LOGGER.info("Nessie Server port: {}", getServicePort(NESSIE_SERVICE_NAME, NESSIE_SERVER_PORT));
+    LOGGER.info("Minio port: {}", getServicePort(MINIO_SERVICE_NAME, MINIO_PORT));
+    LOGGER.info("Nessie Server docker-compose startup cost: {} ms", System.currentTimeMillis() - startTime);
+  }
+
+  public String s3Endpoint() {
+    return "http://localhost:" + getServicePort(MINIO_SERVICE_NAME, MINIO_PORT);
+  }
+
+  public String nessieServerUri() {
+    return "http://localhost:" + getServicePort(NESSIE_SERVICE_NAME, NESSIE_SERVER_PORT);
```

Comment:
This will not work. 

Nessie Uri need a suffix like /api/v1
https://iceberg.apache.org/docs/latest/nessie/#nessie-catalog